x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/advanced_step/advanced_step_form.tsx

```diff
@@ -136,7 +132,7 @@ export const AdvancedStepForm: FC<CreateAnalyticsStepProps> = ({
   const {
     services: { docLinks },
   } = useMlKibana();
-  const { ELASTIC_WEBSITE_URL, DOC_LINK_VERSION } = docLinks;
+  const docLink = docLinks.links.ml.outlierDetectionRoc;
```

Comment:
Why is this link https://www.elastic.co/guide/en/machine-learning/master/ml-dfanalytics-evaluate.html#ml-dfanalytics-roc stored against the key `outlierDetectionRoc`? Isn't the ROC curve associated more with classification jobs? (`outlierDetectionRoc` key was added in https://github.com/elastic/kibana/pull/85365).

Reply:
Looks ROC is listed for outlier detection (https://www.elastic.co/guide/en/machine-learning/master/ml-dfanalytics-evaluate.html#ml-dfanalytics-outlier-detection) and AUC ROC is listed for classification (https://www.elastic.co/guide/en/machine-learning/master/ml-dfanalytics-evaluate.html#ml-dfanalytics-classification).  Should this code use the latter?

Reply:
Or rather, https://www.elastic.co/guide/en/machine-learning/master/ml-dfanalytics-evaluate.html#ml-dfanalytics-class-aucroc

Reply:
Yes in this case we should use https://www.elastic.co/guide/en/machine-learning/master/ml-dfanalytics-evaluate.html#ml-dfanalytics-class-aucroc as this link is shown for the Top classes field for classification jobs:

![image](https://user-images.githubusercontent.com/7405507/102806511-ef20a380-43b4-11eb-8d40-ce33592c3bd3.png)

Reply:
You should also rename that `docLink` variable to something like `classAucRocDocLink` to make it clear which doc page it is referring to.

Reply:
Done, thanks!